### PR TITLE
docs: update TRG 2.5 - draft version of new metadata option repoCategory

### PR DESCRIPTION
**Pull request by tomaszbarwicki**
PR to crate TRG 2.5  draft version with an update of new metadata repoCategory option in .tractusx.

Updates https://github.com/eclipse-tractusx/sig-infra/issues/317

### CODE REVIEW

File: docs/release/trg-0/trg-2-5.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+In the root of a sub/non-leading repository create a file called `.tractusx`:
+
+```yaml
+leadingRepository: "LEADING_PRODUCT_REPOSITORY_URL"
+```
+
+There is an option to specify repository category, it decides which quality checks dashboard section will be used to present repository.
+The field is optional and there are currently two options supported "special" and "support".
+"Special" is dedicated mostly to repositories described as Special Interest Group (repo name begins with "sig-").
```
Comment (FaGru3n) [inline docs/release/trg-0/trg-2-5.md:46]: Hi @tomaszbarwicki, I would use the same field descriptions.

If you start with "special"  then also use "special" in further descriptions or just a little point list:

- "special" is dedicated mostly to repositories described as Special Interest Group (repo name begins with "sig-")
- "support" for repositories which are neither products nor SIG but play vital role in overall ecosystem

**Review by FaGru3n — CHANGES_REQUESTED** [review]
just a either capital letters or small ones to describe the options for "repoCategory"

**Comment by FaGru3n** [comment]
Hi @tomaszbarwicki guess your PR is outdated, because of the current infromation that are presented on https://eclipse-tractusx.github.io/docs/release/trg-2/trg-2-5

**Comment by tomaszbarwicki** [comment]
> Hi @tomaszbarwicki guess your PR is outdated, because of the current infromation that are presented on https://eclipse-tractusx.github.io/docs/release/trg-2/trg-2-5

That seems to be correct, closing this one.

**Pull request closed without merging**